tests/test_all_fields.py

```diff
@@ -12,6 +42,13 @@ class AllFields(TestLinkedinAdsBase):
     def name():
         return "tap_tester_linkedin_ads_all_fields"
 
+    def get_properties(self, original: bool = True):
+        return  {
+            "start_date" : "2018-08-01T00:00:00Z",
+            "accounts": os.getenv("TAP_LINKEDIN_ADS_ACCOUNTS"),
+            "page_size": 100
+        }
+
```

Comment:
Also if we just wanna change the start date we can change the self.start_date and use `connections.ensure_connection(self, original_properties=False)` to use it